config/enabled.yml

```diff
@@ -1173,6 +1173,11 @@ Performance/Count:
                   Use `count` instead of `select...size`, `reject...size`,
                   `select...count`, `reject...count`, `select...length`,
                   and `reject...length`.
+  # This cop has known compatibility issues with Rails. Rails' `count` ignores
+  # a the block that is passed to it. For more information, see the
+  # documentation in the cop itself. If you understand the known risk,
+  # you can enable `RunWithRails`.
+  RunWithRails: false
```

Comment:
I'm not sure I like this name. I'm guessing it's a reference to the Rails cops, but we can probably find better wording here.